cockroachcloud/backups-page.md

```diff
@@ -236,3 +258,17 @@ Or [change the existing table's name](../{{site.versions["stable"]}}/rename-tabl
 ~~~ sql
 > ALTER TABLE target_database.example_table RENAME TO target_database.archived_example_table;
 ~~~
+
+</div>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+## Restore a cluster
+
+Find the cluster backup you want to restore, and click **Restore**.
+
+Performing a restore will cause your cluster to be unavailable for the duration of the restore. All current data is deleted, and the cluster will be restored to the state it was in at the time of the backup. There is no automatic incremental backups, and no automatic database or table level backups.
+
+You can [manage your own backups](run-bulk-operations.html), including incremental, database, and table level backups. To perform manual backups, you must configure either a [`userfile`](../{{site.versions["stable"]}}/use-userfile-for-bulk-operations.html) location or a [cloud storage location](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html), which requires [billing information](billing-management.html) for your organization.
```

Comment:
Minor nit: Can we add this phrase here as well:  "Once you enter billing information, even if you don't set a spend limit, ... "

Reply:
Done.